src/Player.js

```diff
@@ -6,11 +10,29 @@ class Player {
 
   solvePuzzle() {
 
+  }
+  
+  getPlayerScore() {
+
   }
 
   spinWheel() {
     
   }
+
+  buyVowel() {
+    if(this.score > 100) {
+      //buy vowel becomes enabled
```

Comment:
Is they player going to be aware of the condition to get the button undisabled?

Reply:
By that I mean, will there be instructions or a message that says they can't afford it?
Otherwise they might think it's a bug.